### ISSUE TITLE
Plugin with id 'com.github.hierynomus.license-base' not found

### ISSUE THREAD

**Issue by arulrajnet** [hierynomus/license-gradle-plugin#139]
I am using license-gradle-plugin 0.14.0. Gradle version 3.5. Java 1.8

My gradle license config

```
buildscript {
  repositories.maven { url 'https://plugins.gradle.org/m2/' }
  dependencies.classpath 'gradle.plugin.nl.javadude.gradle.plugins:license-gradle-plugin:0.14.0'
}

apply plugin: nl.javadude.gradle.plugins.license.LicensePlugin

license {
    include "**/*.java"
    header rootProject.file('gradle/license.template')
    strictCheck true
    skipExistingHeaders false
    ext {
      copyright_range = Calendar.getInstance().get(Calendar.YEAR)
      name = "Arulraj Venni"
    }
    mapping {
      java = 'DOUBLESLASH_STYLE'
    }
}

```
The error I got
```
* What went wrong:
A problem occurred evaluating script.
> Failed to apply plugin [class 'nl.javadude.gradle.plugins.license.LicensePlugin']
   > Plugin with id 'com.github.hierynomus.license-base' not found.
```

Version 0.13.1 is working fine for the same above config.

**Comment by arulrajnet** [comment]
For your reference this is that demo [repo](https://github.com/arulrajnet/gradle-java-template) . I have used [license.gradle](https://github.com/arulrajnet/gradle-java-template/blob/master/gradle/license.gradle) as external buildscript. 

To reproduce that issue change that version to `0.14.0` in license.gradle.

**Comment by hierynomus** [comment]
Did you follow the README?

https://github.com/hierynomus/license-gradle-plugin/blob/master/README.adoc#source-file-license-application

Looking at your gradle files you're trying to apply the class file, instead
of the plugin.

2017-06-12 19:36 GMT+02:00 Arul <notifications@github.com>:

**Comment by arulrajnet** [comment]
I am using this as external build script. So giving plugin id will not work. That's why I have used class file. Refer http://mrhaki.blogspot.in/2015/10/gradle-goodness-apply-external-script.html

**Comment by hierynomus** [comment]
The "problem" in this case is that the LicensePlugin class tries to apply the base plugin because it needs that.

**Comment by pgelinas** [comment]
If the LicensePlugin applies the two plugins by its class name instead of the id, wouldn't that solve the issue of external script usage?

**Comment by arulrajnet** [comment]
@pgelinas yes its solves that issue. Gave PR for the same. @hierynomus please review

**Comment by To-da** [comment]
@hierynomus are you going to release this fix anytime soon (0.15.1) ? @arulrajnet or is there any workaround for external build script usage?
Thank you.

**Comment by arulrajnet** [comment]
@To-da Move to 0.13.1 version.
